massa-graph/src/block_graph.rs

```diff
@@ -1324,18 +1366,27 @@ impl BlockGraph {
         if self.genesis_hashes.contains(&block_id) {
             return Ok(());
         }
-        debug!(
-            "received block {} for slot {}",
-            block_id, block.header.content.slot
-        );
-        massa_trace!("consensus.block_graph.incoming_block", {"block_id": block_id, "block": block});
+
+        let slot = {
```

Comment:
if this whole retrieval is just there to get the slot, we should add the slot as a parameter of the incoming_block function

Reply:
Yes, retrieve removed and slot added in the ReceiveBlock message.